extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -34,25 +36,43 @@ export class ServerPropertiesDialog extends ObjectManagementDialogBase<Server, S
 	private versionInput: azdata.InputBoxComponent;
 
 	private memoryTab: azdata.Tab;
+	private readonly memoryTabId: string = 'memoryId';
 	private memorySection: azdata.GroupContainer;
 	private minServerMemoryInput: azdata.InputBoxComponent;
 	private maxServerMemoryInput: azdata.InputBoxComponent;
 	private engineEdition: azdata.DatabaseEngineEdition;
 
+	private activeTabId: string;
+	private toDispose: vscode.Disposable[] = [];
```

Comment:
Oh wait, we already have this in the base class. So just remove this and add everything to `this.disposables` and it'll be handled for you.

https://github.com/Microsoft/azuredatastudio/blob/a52f86bc48559413fd34248e4d49480b4787cc9b/extensions/mssql/src/ui/dialogBase.ts#L109